registration/models.py

```diff
@@ -79,7 +79,7 @@ class BetaTestApplication(ValidateModelMixin, TimeStampedModel):
 
     user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
     subdomain = models.CharField(
-        max_length=255,
+        max_length=63,
```

Comment:
Why 63?  (I haven't read the RFC).

Have you checked to see if any of the existing instances' subdomains exceed 63 characters?

Since this change will actually cause the database field to be truncated for all instances (be they archived or not), it's probably best to avoid changing the actual `max_length` and instead just add another validator.

Reply:
I did do a check of all the instances. I exported all instances using the API and used a Python script to validate the maximum length and the regex. No subdomains were too long, but a couple failed validation. Those were attached to the ticket. 

I will change this to a validator though, as that will also give us better control over the error message when validation fails.